src/stories/blocks/CardFooter.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import CardFooter from 'components/blocks/CardFooter';
+
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Blocks/Molecules/CardFooter',
+  component: CardFooter,
+  decorators: [
+    (Story) => (
+      <div className="container">
+        <Story />
+      </div>
+    ),
+  ],
+} as ComponentMeta<typeof CardFooter>;
+
+const Template: ComponentStory<typeof CardFooter> = (args) => (
+  <CardFooter {...args} />
+);
+
+export const NotMuted = Template.bind({});
+NotMuted.args = {
+  attributes: {
+    content: 'This is a Card Footer',
+    muted: false,
```

Comment:
I believe `muted` is unnecessary here, since it's not used in `CardFooter.tsx` (or the corresponding `save.js` file in the Plugin repo).

Reply:
@MichaelAllenWarner – I removed the single-quotes and also removed the muted attribute.